src/ui/js/Game.js

```diff
@@ -1917,20 +1918,39 @@ Game.pageAddSkillListFooter = function() {
       skillDesc += interactDescArray.join('\n');
     }
 
-    skillSpan = $('<span>').append($('<span>', {
+    skillSpan = $('<span>', {
       'text': skill,
-      'title': skillDesc,
       'class': 'skill_desc',
-    })).append($('<span>', {
-      'text': 'i',
-      'title': skillDesc,
-      'class': 'info_icon',
-    }));
+    });
+
+    tooltipDiv = $('<div>', {
+      'text': skillDesc,
+      'class': 'hidden_tooltip',
+    });
+
+    skillSpan.on({
+      'click': function() {
+          if ($(this).data('ui-tooltip')) {
+            $(this).tooltip('destroy');
+          } else {
+            $(this).tooltip({
+              items: '.skill_desc',
+              content: $(this).next('.hidden_tooltip').html(),
+            });
+            $(this).tooltip('open');
+          }
+        },
+      'mouseout': function(event) {
+        event.stopImmediatePropagation();
+      }
+    });
 
     if (info.code) {
       dieSkillSpanArray.push(skillSpan);
+      dieSkillSpanArray.push(tooltipDiv);
```

Comment:
This seems weird at a glance, like you have an array of skillSpan, tooltipDiv, skillSpan, tooltipDiv, as opposed to each tooltipDiv being in some way associated with its corresponding skillSpan.  I'll look at the HTML in testing.